src/AaveV2_Eth_ServiceProviders_20231907/TestTrade.s.sol

```diff
@@ -48,6 +50,6 @@ contract TestTrade is Script {
 
     data = abi.encode(paths, reverses);
 
-    return abi.encode(2000, data); // 2000 = 20% slippage just cause it's a super small amount
+    return abi.encode(3500, data); // 2000 = 20% slippage just cause it's a super small amount
```

Comment:
Seems this changed for testing? Comment should probably be updated, anyway why is the slippage so high for testing, gas concerns?

Reply:
100 USDC is not much unfortunately, so need to do a big spread for it to be picked up and get killed on the slippage 'cause if not it doesn't even cover gas costs.

Reply:
It's not so much that it's only say going from 100 USDC to 99.5 aUSDC or whatever, the slippage tolerance has to account for gas costs also so that's why it ends up being bigger.

Reply:
Fair enough, got it!